maps/_common/areas/shuttles.dm

```diff
@@ -205,11 +205,15 @@
 
 /area/shuttle/research/station
 	icon_state = "shuttle2"
+	flags = RAD_SHIELDED | SPAWN_ROOF
+	base_turf = /turf/simulated/floor/asteroid/ash
 	station_area = 1
 
 /area/shuttle/research/outpost
 	icon_state = "shuttle"
-	station_area = 1
+	station_area = 0
```

Comment:
Compatibility wise it would probs be better to have a `/area/shuttle/research/away` and use that for the away missions while leaving the `/area/research/outpost` as is